tests/e2e/specs/modules/analytics/write-scope-requests.test.js

```diff
@@ -342,17 +342,17 @@ describe( 'Analytics write scope requests', () => {
 
 		interceptCreatePropertyRequest = true;
 
-		await expect( page ).toClick( '.mdc-dialog--open .mdc-button', {
-			text: /proceed/i,
-		} );
-
-		// expect( console ).toHaveErrored(); // Permission scope error.
-		await page.waitForRequest( ( req ) =>
-			req.url().match( 'analytics-4/data/create-property' )
-		);
-		await page.waitForRequest( ( req ) =>
-			req.url().match( 'analytics-4/data/create-webdatastream' )
-		);
+		await Promise.all( [
```

Comment:
Note that I've applied the same approach to the other active test in the file, for consistency and to avoid potential flakiness in this test.